lib/tasks/javascript.rake

```diff
@@ -73,7 +73,6 @@ def dependencies
     },
     { source: "chart.js/dist/chart.min.js", public: true },
     { source: "chartjs-plugin-datalabels/dist/chartjs-plugin-datalabels.min.js", public: true },
-    { source: "diffhtml/dist/diffhtml.min.js", public: true },
```

Comment:
`diffhtml` is no longer vendored.

Reply:
Have we updated all plugins that were using it? (just discourse-ai?)

I wonder if we need to defer 'dropping diffhtml' to a later PR?